internal/clientsholder/clientsholder.go

```diff
@@ -212,6 +216,20 @@ func newClientsHolder(filenames ...string) (*ClientsHolder, error) { //nolint:fu
 	if err != nil {
 		return nil, fmt.Errorf("cannot instantiate k8s networking client: %s", err)
 	}
+	discoveryClient, _ := discovery.NewDiscoveryClientForConfig(clientsHolder.RestConfig)
```

Comment:
Should we bake the `discoveryClient` into the `clientsholder` similar to how we do the `K8sClient`, etc?

Reply:
im not sure if we need it to another thing